File: 混合开发/Objective-C项目中调用Swift/Objective-C项目中调用Swift/ViewController.m
//
//  ViewController.m
//  Objective-C项目中调用Swift
//
//  Created by 杨帆 on 2019/7/18.
//  Copyright © 2019 杨帆. All rights reserved.
//

#import "ViewController.h"
#import <Objective_C项目中调用Swift-Swift.h>

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
    
    Person *person = [[Person alloc]init];
    person.name = @"YungFan";
    [person eat];
}


@end

File: 混合开发/Swift项目中调用Objective-C/Swift项目中调用Objective-C/ViewController.swift
//
//  ViewController.swift
//  Swift项目中调用Objective-C
//
//  Created by 杨帆 on 2019/7/18.
//  Copyright © 2019 杨帆. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.
        
        //直接像实用Swift对象一样使用OC对象
        let person = Person()
        person.name = "YungFan"
        person.eat()
    }


}

